x-pack/plugins/security/server/plugin.ts

```diff
@@ -216,6 +248,13 @@ export class Plugin {
       this.securityLicenseService = undefined;
     }
 
+    if (this.featureUsageService) {
+      this.featureUsageService = undefined;
+    }
+
+    if (this.featureUsageServiceStart) {
+      this.featureUsageServiceStart = undefined;
+    }
```

Comment:
I know you are also doing it with your other services (but the other service also have a call to a destroy-ish method), but are these `undefined` attributions really useful in any way?

Reply:
No I suppose they aren't. I can at least get rid of the logic for `featureUsageService`, but I'll probably keep the handling for `featureUsageServiceStart` for the time being